store.json

```diff
@@ -221,5 +221,18 @@
     "labels": {
       "com.openfaas.ui.ext": "jpg"
     }
+  },
+  {
+    "title": "Awesomify",
+    "description": "Make anyone or anything awesome!",
```

Comment:
I'm not sure that this meme or tool is widely-known. It may benefit from being more descriptive in the entry.

Reply:
I've switched the description to a much longer and more explanatory one. Can you give it another look to see if it's more suitable? Thanks.